toolkit/docs/building/prerequisites.md

```diff
@@ -1,28 +1,9 @@
 
 # Build Requirements
 
-## Requirements were validated on `Ubuntu 18.04`
+## CBL-Mariner
```

Comment:
Please surround headings with blank lines. See [here](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022). Id you're using VS Code, try installing [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to see and fix similar issues more easily.

Reply:
Thanks for you advice.
Done!